CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Release 1.0.1 (2017-08-25)
+
+## Major Themes
+
+Algorithms for applying northbound configuration (stored in ETCD key value data store)
+to VPP in proper order of VPP binary API calls implemented in [Default VPP plugin](plugins/defaultplugins): 
+- network interfaces, especially:
+  - MEMIFs (optimized dataplane network interface tailored for container to container network connectivity)
+  - VETHs (standard Linux Virtual Ethernet network interface)
+  - AF_Packets (for accessing VETHs and similar type of interface)
+  - VXLANs, Physical Network Interfaces, loopbacks ...
+- L2 BD & X-Connects
+- L3 IP Routes & VRFs
+- ACL (Access Control List)
+
+Support for Linux VETH northbound configuration implemented in [Linux Plugin](plugins/linuxplugin)
+applied in proper order with VPP AF_Packet configuration.
+
+Data Synchronization during startup for network interfaces & L2 BD
+(support for situation when ETCD contain configuration before VPP Agent starts).
+
+Data replication and events:
+- Updating operational data in ETCD (VPP indexes such as  sw_if_index) and statistics (port counters).
+- Updating statistics in Redis (optional once redis.conf available - see flag in FlavorRedis).
+- Publishing link up/down events to Kafka message bus.
+
+Miscellaneous:
+- [Examples](examples)
+- Tools:
+  - [agentctl CLI tool](cmd/agentctl) that show state & configuration of VPP agents
+  - [docker](docker): container-based development environment for the VPP agent
+- other features inherited from cn-infra:
+ - [health](https://github.com/ligato/cn-infra/tree/master/health): status check & k8s HTTP/REST probes
+ - [logging](https://github.com/ligato/cn-infra/tree/master/logging): changing log level at runtime
+
+### Extensibility & integration
+Ability extend behavior of the VPP Agent by creating new plugins on top of [VPP Agent flavor](flavors/vpp).
```

Comment:
Ability to extend the behavior of the VPP Agent by creating new plugins on top of [VPP Agent flavors](flavors/vpp)